tools/travis/run.sh

```diff
@@ -2,7 +2,8 @@
 set -e
 
 # Short circuit tests and linting jobs if there are no code changes involved.
-if [[ $TOXENV != docs ]]; then
+if [[ $TOXENV != docs && $TOXENV != lint-py2 && $TOXENV != lint-py3 ]]; then
```

Comment:
Good move -- we should probably also fix the line length issue in this PR.

Reply:
Yup, just checking that it is failing as expected :)

Reply:
> Yup, just checking that it is failing as expected :)

Oh, this file being modified will trigger all checks in the CI.